github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```
@@ -576,3 +576,45 @@ periodics:
       - name: token
         secret:
           secretName: oauth-token
+
+- name: periodic-project-infra-mirror-istioctl
+  cron: "25 1 * * 1"
+  decorate: true
+  annotations:
+      testgrid-create-test-group: "false"
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  extra_refs:
+  - org: kubevirt
+    repo: project-infra
+    base_ref: main
+    workdir: true
```

Let's have a consistent indentation for all the array elements in the job definition. 
```suggestion
    - org: kubevirt
      repo: project-infra
      base_ref: main
      workdir: true
```